vars/warnAboutDeprecatedPostgres.groovy

```diff
@@ -10,6 +10,12 @@ switch (gitUrl.toLowerCase()) {
     case "https://github.com/hmcts/am-org-role-mapping-service.git":
     case "https://github.com/hmcts/am-role-assignment-service.git":
     case "https://github.com/hmcts/am-judicial-booking-service.git":
+    case "https://github.com/hmcts/ccd-data-store-api":
```

Comment:
URL needs the .git on the end Eg: "https://github.com/hmcts/ccd-data-store-api.git"